uber/decorators.py

```diff
@@ -529,6 +529,18 @@ def set_renderable(func, access):
     return new_func
 
 
+def only_renderable(*needs_access):
```

Comment:
This is a bit nitpicky, but even though it follows the naming convention of `all_renderable` I don't think `only_renderable` really communicates what this function does. How about something like `renderable_override`? I can see why that name might not be desirable either, so if you don't like it we'll just stick to `only_renderable`.

Reply:
Yeah, I didn't really like it either, but I was trying make it similar to `all_renderable`.

I like `renderable_override` much more. It makes the intent of the function much clearer 😄